cmd/statusd/main.go

```diff
@@ -1,174 +1,134 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"log"
 	"os"
-	"path/filepath"
 	"runtime"
+	"strings"
 
 	"github.com/status-im/status-go/geth/api"
 	"github.com/status-im/status-go/geth/params"
-	"gopkg.in/urfave/cli.v1"
 )
 
 var (
-	gitCommit  = "rely on linker: -ldflags -X main.GitCommit"
-	buildStamp = "rely on linker: -ldflags -X main.buildStamp"
-	app        = makeApp(gitCommit)
-	statusAPI  = api.NewStatusAPI()
+	gitCommit  = "N/A" // rely on linker: -ldflags -X main.GitCommit"
+	buildStamp = "N/A" // rely on linker: -ldflags -X main.buildStamp"
 )
 
 var (
-	// ProdModeFlag is whether we need dev or production settings
-	ProdModeFlag = cli.BoolFlag{
-		Name:  "production",
-		Usage: "Whether production settings should be loaded",
-	}
+	prodMode       = flag.Bool("production", false, "Whether production settings should be loaded")
+	nodeKeyFile    = flag.String("nodekey", "", "P2P node key file (private key)")
+	dataDir        = flag.String("datadir", params.DataDir, "Data directory for the databases and keystore")
+	networkID      = flag.Int("networkid", params.RopstenNetworkID, "Network identifier (integer, 1=Homestead, 3=Ropsten, 4=Rinkeby, 777=StatusChain)")
+	whisperEnabled = flag.Bool("shh", false, "SHH protocol enabled")
+	swarmEnabled   = flag.Bool("swarm", false, "Swarm protocol enabled")
+	httpEnabled    = flag.Bool("http", false, "HTTP RPC enpoint enabled (default: false)")
+	httpPort       = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
+	ipcEnabled     = flag.Bool("ipc", false, "IPC RPC enpoint enabled")
+	logLevel       = flag.String("log", "", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
+	logFile        = flag.String("logfile", "", "Path to the log file")
+	version        = flag.Bool("version", false, "Print version")
+	verbose        = flag.Bool("v", false, "Be verbose to stdout (overrides -logLevel and -logFile)")
+)
 
-	// NodeKeyFileFlag is a node key file to be used as node's private key
-	NodeKeyFileFlag = cli.StringFlag{
-		Name:  "nodekey",
-		Usage: "P2P node key file (private key)",
-	}
+func main() {
+	flag.Parse()
 
-	// DataDirFlag defines data directory for the node
-	DataDirFlag = cli.StringFlag{
-		Name:  "datadir",
-		Usage: "Data directory for the databases and keystore",
-		Value: params.DataDir,
+	config, err := makeNodeConfig()
+	if err != nil {
+		log.Fatalf("Making config failed: %v", err)
+		return
```

Comment:
No return needed. log.Fatal(f) calls os.Exit()

Reply:
That's true, but I was changing this a couple of times to `fmt.Println()` and back, while refactoring code, so decided it'll be safer to leave return to make clear that we want to end here, no matter how we log it.